packages/discord.js/src/structures/Message.js

```diff
@@ -357,6 +357,16 @@ class Message extends Base {
     } else {
       this.interaction ??= null;
     }
+
+    if (typeof data.position !== 'undefined') {
```

Comment:
Actually, should be like this to be consistent with the rest of the properties
```suggestion
    if ('positon' in data) {
```